app/views/content/is-teaching-right-for-me/languages/_article.html.erb

```diff
@@ -0,0 +1,105 @@
+<div class="row inset">
+  <section class="col col-720">
+    <p>A modern languages teacher can  be a rewarding way to use your language knowledge, while opening young minds to the languages and cultures around the world.</p>
+  <p>Teacher training is available in 18 modern languages with the most popular being French, German and Spanish in key stage 2 and 3.</p>
+  <p>Tax-free  bursaries of £25,000 or scholarships of £27,000 are available for eligible trainee modern languages teachers.</p>
+  </section>
+
+  <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithTwoImagesComponent.new(
+      heading: "Empower your pupils",
+      colour: "pink",
+      image_1_path: "static/images/content/hero-images/0017.jpg",
```

Comment:
@caitlin-reidy can you or Jen swap all the images for the new language teacher ones we have from the campaign

Reply:
@jenhadfield-dfe are you able to change the images please